salt/core/sshd/init.sls

```diff
@@ -1,9 +1,19 @@
 # We'll only be using SSH key authentication.
+{% if grains['osrelease']|float >= 20.04 %}
+/etc/ssh/sshd_config.d/customization.conf:
+  file.managed:
+    - source: salt://core/sshd/files/customization.conf
+    - watch_in:
+      - service: ssh_service
+
+{% else %}
```

Comment:
Could we instead add `Include /etc/ssh/sshd_config.d/*.conf` to `/etc/ssh/sshd_config` if not present, and then use the same Salt instructions for both Ubuntu versions?

Reply:
Unfortunately `Include` isn't supported on Ubuntu 18 instances in `sshd_config` (sshd) - it can be used in `ssh_config` but that is for a different purpose.

Reply:
Oops, I checked the wrong man page. Sounds good then.